README.md

```diff
@@ -554,6 +554,33 @@ This is the expected result after removing preferences with the URL code specify
 }
 ```
 
+### https://[APPURL]/api/Terminology/[id]/standard
```

Comment:
Not sure what we want the endpoint to be.

Also,
I use `standard_terminology` to define the references throughout this PR.
Let me know if we wanted different language surrounding this entirely. Should `standard_terminology` be something else (`prefered_terminology`) ?